src/components/package/_fields/custom-coverage/package-coverage-fields.js

```diff
@@ -2,39 +2,55 @@ import React, { Component, Fragment } from 'react';
 import { Field, FieldArray } from 'redux-form';
 import PropTypes from 'prop-types';
 import moment from 'moment';
-import { FormattedMessage, injectIntl, intlShape } from 'react-intl';
+import {
+  FormattedMessage,
+  injectIntl,
+  intlShape,
+} from 'react-intl';
 
 import {
   Datepicker,
-  RepeatableField
+  RepeatableField,
 } from '@folio/stripes/components';
 
 import styles from './package-coverage-fields.css';
 
 class PackageCoverageFields extends Component {
   static propTypes = {
     initialValue: PropTypes.array,
-    intl: intlShape
+    intl: intlShape,
   };
 
   static defaultProps = {
-    initialValue: []
+    initialValue: [],
   };
 
+  constructor(props) {
+    super(props);
+
+    this.state = { coverageDateAmount: 1 };
+  }
+
   validateCoverageDate = (value) => {
     const { intl } = this.props;
     moment.locale(intl.locale);
     let dateFormat = moment.localeData()._longDateFormat.L;
     let errors;
 
     if (value && !moment.utc(value).isValid()) {
-      errors = <FormattedMessage id="ui-eholdings.validate.errors.dateRange.format" values={{ dateFormat }} />;
+      errors = (
+        <FormattedMessage
+          id="ui-eholdings.validate.errors.dateRange.format"
+          values={{ dateFormat }}
+        />);
     }
 
     return errors;
   }
 
   renderField = (dateRange) => {
+    const formatField = (value) => (value ? moment.utc(value) : '');
+
```

Comment:
There is no need to wrap a single parameter with parentheses. The same about the function's body here

Reply:
@MaksymDryha, Unfortunately ESLint does not allow me to remove parentheses from the body.